src/nnet3/nnet-example.cc

```diff
@@ -56,23 +56,25 @@ bool NnetIo::operator == (const NnetIo &other) const {
 }
 
 NnetIo::NnetIo(const std::string &name,
-               int32 t_begin, const MatrixBase<BaseFloat> &feats):
+               int32 t_begin, const MatrixBase<BaseFloat> &feats,
+               int32 skip_frame):
     name(name), features(feats) {
-  int32 num_rows = feats.NumRows();
-  KALDI_ASSERT(num_rows > 0);
-  indexes.resize(num_rows);  // sets all n,t,x to zeros.
-  for (int32 i = 0; i < num_rows; i++)
-    indexes[i].t = t_begin + i;
+  int32 num_skipped_rows = feats.NumRows();
```

Comment:
I don't like the fact that you changed num_rows to num_skipped_rows here.  
It's really not clear what the meaning is supposed to be.
Maybe if you changed 'skip_frame' to 'frame_subsampling_factor' it would be clear?   And changed back num_skipped_rows to num_rows?